tests/unit/plugins/connection/test_docker.py

```diff
@@ -41,6 +41,7 @@ def setUp(self):
     def tearDown(self):
         pass
 
+    @mock.patch('ansible_collections.community.docker.plugins.connection.docker._load_name', 'community.docker.docker')
```

Comment:
You might need to do `from ansible_collections.community.docker.plugins.connection import docker as docker_plugin` at the top and then `docker_plugin._load_name = 'community.docker.docker'`. `@mock.patch()` only wants to change properties that already exist.